docker/arbitrum/arbitrum.sh

```diff
@@ -52,18 +52,25 @@ wait_for_sequencer() {
     done
 }
 
-fund_account() {
-    ${TEST_NODE_SCRIPT} script send-l1 --ethamount 100 --to address_${ADDRESS}
-    ${TEST_NODE_SCRIPT} script send-l2 --ethamount 100 --to address_${ADDRESS}
-    # Wait a bit for the transactions to go through
-    sleep 3
+configure_repo() {
+    repo="https://github.com/OffchainLabs/nitro.git"
+    commit_id="0b32740ddd245ff0e52b1fd1a0372b90195f6a0c"
+    git init ${NITRO}
+    cd ${NITRO}
+    git fetch --depth 1 ${repo} ${commit_id} 
+    git checkout ${commit_id}
+    git submodule update --init blockscout
 }
 
+if [ ! -d ${NITRO} ]; then
+    configure_repo
+fi
+patch_test_node_script
+
 up() {
     echo Starting the end-to-end environment
-    ${TEST_NODE_SCRIPT} --init --detach --no-blockscout
+    ${TEST_NODE_SCRIPT} --init --detach
```

Comment:
Why remove the `--no-blockscout` option?

Reply:
because there is no --no-blockscout option anymore. By default blockscout is not necessary, you can now opt in to turn it on.

Reply:
no option like this now